slo-monitor/slo-monitor-pod.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
```

Comment:
Why do we need both:
- deployment
- pod & service?

Can we just leave one of those (i.e. deployment)?